pkg/entities/record.go

```diff
@@ -38,15 +38,18 @@ type Record interface {
 	GetTemplateID() uint16
 	GetFieldCount() uint16
 	GetInfoElements() []*InfoElementWithValue
+	GetInfoElement(name string) *InfoElementWithValue
+	ContainsInfoElement(name string) bool
```

Comment:
There are pros and cons. Providing a `ContainsInfoElement` method will make it easier when we check whether many info elements are in the record (e.g. for [getting 5-tuple info](https://github.com/vmware/go-ipfix/blob/b51831a0ba8ea36c72c24cbff14a65610a2ba060/pkg/intermediate/aggregate.go#L178)) because checking whether the key is in the map will return two values:
```
if val, exist := map["key"]; ok{
   dosomething()
}
```
But it only works well specially for this case, if users want to read the map in other ways, two functions are not that good. Which way do you prefer?

Reply:
How about having one function to get the map and have couple of simple utilities to get the element and check whether element exists or not?